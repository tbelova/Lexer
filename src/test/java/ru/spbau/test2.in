  x := 0;
 z:= 1;
	t := 2;
