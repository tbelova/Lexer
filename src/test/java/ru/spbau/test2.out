PROGRAM((1,2)(4,4))
   STATEMENTS((1,2)(3,7))
      STATEMENT((1,2)(1,8))
         ASSIGNMENT((1,2)(1,7))
            IDENT((1,2)(1,2))[x]
            EXPRESSION((1,7)(1,7))
               INTEGER((1,7)(1,7))[0]
      STATEMENTS((2,1)(3,7))
         STATEMENT((2,1)(2,6))
            ASSIGNMENT((2,1)(2,5))
               IDENT((2,1)(2,1))[z]
               EXPRESSION((2,5)(2,5))
                  INTEGER((2,5)(2,5))[1]
         STATEMENTS((3,1)(3,7))
            STATEMENT((3,1)(3,7))
               ASSIGNMENT((3,1)(3,6))
                  IDENT((3,1)(3,1))[t]
                  EXPRESSION((3,6)(3,6))
                     INTEGER((3,6)(3,6))[2]
